client/layout/index.jsx

```diff
@@ -117,7 +119,9 @@ Layout = React.createClass( {
 			<MasterbarLoggedIn
 				user={ this.props.user }
 				section={ this.props.section.group }
-				sites={ this.props.sites } />
+				sites={ this.props.sites }
+				setNextLayoutFocus={ this.props.setNextLayoutFocus }
```

Comment:
Why not just `connect` `MasterbarLoggedIn` and allow it to call `setNextLayoutFocus` via its own `mapDispatchToProps` ?